Topics/Software_Engineering/XP.md

```diff
@@ -0,0 +1,100 @@
+# Intro to XP
+
```

Comment:
Consider adding a table of contents here so the reader knows what topics this page covers at a glance.